circle.yml

```diff
@@ -1,41 +1,99 @@
-general:
-  artifacts:
-  - "target/test.log"
-  - "gc.log"
+defaults:
+  - &defaults
+    working_directory: ~/kafka-serialisation
+    docker:
+      - image: olib963/sbt-ci-image:sbt-1-1-5
```

Comment:
I'd be cautious about using private images in CircleCI builds.

Reply:
It's public if that helps? Or by private do you just mean "my own image"?

Reply:
Why does the build pipeline need the docker image?

Reply:
I can switch to using [circleci image](https://hub.docker.com/r/circleci/openjdk/) instead? It just contains java, scala & sbt.

Reply:
As far as I am aware all v2 workflow jobs require an image to run on?

Reply:
The "circleci/openjdk:8-jdk" should be ok